### ISSUE TITLE
Color contrast errors on progress indicators of stepped form

### ISSUE THREAD

**Issue by sarongebre** [unlcms/project-herbie#1088]
Element has insufficient color contrast of 4.4 (foreground color: #777777, background color: #fefdfa, font size: 8.5pt (11.3012px), font weight: normal). Expected contrast ratio of 4.5:1

**Comment by sarongebre** [comment]
Will be fixed on https://github.com/unlcms/project-herbie/issues/1161
